rkt/image/fetcher.go

```diff
@@ -205,12 +205,12 @@ func (f *Fetcher) getRemoteForURL(u *url.URL) (*imagestore.Remote, error) {
 		return nil, nil
 	}
 	urlStr := u.String()
-	if rem, ok, err := f.S.GetRemote(urlStr); err != nil {
+	rem, err := f.S.GetRemote(urlStr)
+	if err != nil && err != imagestore.ErrRemoteNotFound {
```

Comment:
As an example, rlin this case, returning nil for a not found remote won't need the error type check.